api/symphony-api-no-k8s-munchen.json

```diff
@@ -120,7 +120,7 @@
             "name": "activations-manager",
             "type": "managers.symphony.activations",
             "properties": {
-              "providers.state": "k8s-state",
+              "providers.persistentstate": "k8s-state",
```

Comment:
In this json, we actually used memory state provider which should be marked as providers.volatilestate. Similar mismatches occur in other json files. Please help to review and correct.

Reply:
Discussed with @RemindD , this may not be a mismatch. We need to write this config from manager perspective. In manager init function, it will call either `GetPersistentStateProvider` or `GetVolatileStateProvider` to expect deployer to configure the correct state provider. If users use memory state provider in the place where manager expect to use persistent state provider, symphony may lose data when it crashes.

@RemindD , I suggest we can add docs to clarify this. The doc can include,
1. The expectation of state provider for different managers.
2. Which state provider can be used when manager request "providers.persistentstate" or "providers.volatilestate".